main/tasks.py

```diff
@@ -273,3 +277,102 @@ def email_monthly_charges_report() -> None:
     email_monthly_charges_report_logic(
         last_month_start.month, last_month_start.year, last_month_name
     )
+
+
+def sync_clockify_time_entries() -> None:
+    """Task to sync time entries from Clockify API to TimeEntry model."""
+    api_key = os.getenv("CLOCKIFY_API_KEY")
+    if not api_key:
+        print("Clockify API key not found in environment variables")
+        return
+
+    days_back = 30
+    api = ClockifyAPI(api_key)
+    end_date = timezone.now()
+    start_date = end_date - datetime.timedelta(days=days_back)
+
+    project_ids = list(
+        Project.objects.exclude(clockify_id="").values_list("clockify_id", flat=True)
+    )
+
+    total_entries_synced = 0
+    total_entries_skipped = 0
+
+    for project_id in project_ids:
+        try:
+            print(f"Processing project ID: {project_id}")
```

Comment:
Again, use the logger.
```suggestion
            logger.info(f"Processing project ID: {project.clockify_id}")
```